.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI for packaging
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install dependencies
+        run: |
+          sudo apt-get install -y rpm make ruby ruby-dev rubygems build-essential
+          sudo gem install --no-document fpm
+          sudo apt-get install -y rpm
```

Comment:
![image](https://user-images.githubusercontent.com/6814606/108729410-73360980-7565-11eb-82a9-7a1278032073.png)

Reply:
After discussing with @Yiyiyimu, I deleted the test, and the test part can be placed in the CI of APISIX.

Reply:
ok, got it